bridge/github/export.go

```diff
@@ -117,20 +118,38 @@ func (ge *githubExporter) ExportAll(repo *cache.RepoCache, since time.Time) (<-c
 				return
 			}
 
-			snapshot := b.Snapshot()
+			select {
 
-			// ignore issues created before since date
-			// TODO: compare the Lamport time instead of using the unix time
-			if snapshot.CreatedAt.Before(since) {
-				out <- core.NewExportNothing(b.Id(), "bug created before the since date")
-				continue
-			}
+			case <-ctx.Done():
+				if ctx.Err() == nil {
+					return
```

Comment:
hmmm, doesn't that need to return an error of some sort ? Could you try with a https://golang.org/pkg/context/#WithTimeout to see what happen ?

Reply:
it is true that these errors should be handled properly (i didn't pay attention to that at first sight)
I tried to run some tests locally. Two types of errors are catched when context is cancelled (timeout or manual cancel):
 - `context deadline exceeded` for context timeout
- `context cancelled` when cancel() is called

Reply:
Is it really useful to test and return if `ctx.Err() == nil` ? I'm not entirely familiar with `context` but if `ctx.Done()` happen, does it may or have to be an error ?

Reply:
I just checked this. There will be always an error when the context is done. 
there are two possible errors:
- `context deadline exceeded`
- `context cancelled`

Should we consider `context cancelled` as soft error ?

Reply:
Both are basically the same thing, the context is no longer valid. They should have the same consequences.